control-plane-operator/controllers/awsprivatelink/awsprivatelink_controller.go

```diff
@@ -389,6 +389,19 @@ func reconcileAWSEndpointService(ctx context.Context, awsEndpointService *hyperv
 			}
 			return err
 		}
+
+		if aws.StringValue(output.VpcEndpoints[0].ServiceName) != awsEndpointService.Status.EndpointServiceName {
```

Comment:
I don't think this is needed here, only in the case when we don't have an endpoint ID in status

Reply:
I will test the behaviour with this change.

Reply:
Ack, thx. I'm ok with it as is if it works in testing

Reply:
Please elaborate in all these logs what's "conflicting". If I'm understanding correctly I'd say something like
`Current endpoint is not pointing to the existing .Status.EndpointServiceName, reconciling by deleting endpoint`
ANd

Reply:
updated